qa/rpc-tests/wallet_protectcoinbase.py

```diff
@@ -243,6 +243,7 @@ def run_test (self):
         wait_and_assert_operationid_status(self.nodes[0], myopid, "failed", "Insufficient shielded funds, have 9.9998, need 10000.0001")
 
         # Send will fail because of insufficient funds unless sender uses coinbase utxos
+        errorString = ""
```

Comment:
Note: We were not resetting `errorString` and the next check was similar enough to the one above that I thought it was possible for the next assertion to sometimes pass when it should not, so I went ahead and made sure that the string was being reset.